safety.sh

```diff
@@ -0,0 +1,18 @@
+#/bin/bash
+currenttime=`date +"%d-%b-%Y %H:%M:%S"`
+logfile=`date +"%d-%b-%Y"`_log.log
+echo "================= $currenttime"  >> $logfile
+RMLEDGER=1500
+CLUSTERSLOT=$(solana slot -u http://10.142.0.4:8899)
```

Comment:
Can you please turn `10.142.0.4` into a constant, so it's more clear what host this